app/controllers/manifests_controller.rb

```diff
@@ -26,4 +30,14 @@ def index
   def show
     @manifest = Manifest.find_by_uuid_or_tracking_number!(params[:id])
   end
+
+  private
+
+  def validate_manifest(content)
```

Comment:
is it the same? I think in the API controller the validate_manifest method renders a JSON response and in the web controller validate_manifest sets the `@errors` only.